src/net_processing.cpp

```diff
@@ -1132,8 +1147,8 @@ void PeerManagerImpl::RemoveBlockRequest(const uint256& hash, std::optional<Node
         return;
     }
 
-    // Currently we don't request more than one peer for same block
-    Assume(mapBlocksInFlight.count(hash) == 1);
+    // We should never requested too many of this block
```

Comment:
nit: perhaps "We should not have requested too many of this block"

Reply:
done